### PR TITLE
workflows: request a bottle rebuild after a merge

### PR DESCRIPTION
**Pull request by iMichka**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

**Comment by jonchang** [comment]
Will this need to `brew tap homebrew/linux-dev`?

**Comment by iMichka** [comment]
I added that step, it's needed.

I am unsure if any credentials will be needed or if the GitHub token is enough.

**Comment by jonchang** [comment]
The token will need the `repo` permission, I believe, which I think it will have since it is a maintainer pushing the merge commit.

File: .github/workflows/request-bottle-after-merge.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Checkout tap
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup tap
+        run: |
+          rm -rf $(brew --repository ${{github.repository}})
+          ln -s $GITHUB_WORKSPACE $(brew --repository ${{github.repository}})
+      - name: Tap linux-dev
+          brew tap homebrew/linux-dev
```
Comment (dawidd6) [inline .github/workflows/request-bottle-after-merge.yml]: Forgot about `run:`

File: .github/workflows/request-bottle-after-merge.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+jobs:
+  upload-bottles:
+    if: startsWith(github.event.head_commit.message, 'Merge') == true
+    runs-on: ubuntu-latest
+    container:
+      image: homebrew/brew
+    steps:
+      - name: Checkout tap
+        uses: actions/checkout@v2
+        with:
```
Comment (dawidd6) [inline .github/workflows/request-bottle-after-merge.yml]: Is the full history necessary? Wouldn't say so.

**Comment by iMichka** [comment]
Let's see if this works

**Pull request merged**